renderer/HardwareOpenGL.cpp

```diff
@@ -595,15 +597,18 @@ int opengl_Setup(oeApplication *app, int *width, int *height) {
     rend_SetErrorMessage(buffer);
     return (0);
   } // if
-  SDL_SetEventFilter(d3SDLEventFilter);
+  SDL_SetEventFilter(d3SDLEventFilter, NULL);
 
   bool fullscreen = false;
+
+#if 0  // this was a workaround for 3DFx Voodoo cards back in the day. Presumably no longer needed?    --ryan 04/27/2024
```

Comment:
I'd say let's kill it

Reply:
It's gone, as of 6f07fd5a94190f67813d637ffbce77d8507753a7 !